Added check for integers when parsing user input.

alanefl opened the pull request:
Added method "tryParsing" to check if the user input string is an
integer, otherwise display a pane to tell the user to enter an integer.

alanefl committed:
Added check for integers when parsing user input.

Added method "tryParsing" to check if the user input string is an
integer, otherwise display a pane to tell the user to enter an integer.

FOSSLGYS commented on src/com/tapia/foss/frames/Visualizer.java:
```diff
@@ -261,6 +273,21 @@ private void display()
+     *
+     * Returns false if the parsing fails and displays a GUI window to notify the user.
+     */
+    private boolean tryParsing(String input) {
```
How about we rename this to isInteger()

alanefl replied:
Definitely a more descriptive name.  I can pull out the JOptionPane notification from this method to the code that calls it for clarity.

FOSSLGYS commented on src/com/tapia/foss/frames/Visualizer.java:
```diff
@@ -210,8 +210,14 @@ private void jButtonRemoveActionPerformed(java.awt.event.ActionEvent evt) {//GEN
+                        JOptionPane.showMessageDialog(this, "Unable to find/delete "+input);
+                    }
+                } else {
+                    return;
```
Instead of returning, how about we give a message to the user using JOptionPane giving feedback on sending in an integer only.
I dont think return is necessary here, is it?

alanefl replied:
See the comment below.

FOSSLGYS commented on src/com/tapia/foss/frames/Visualizer.java:
```diff
@@ -232,7 +238,13 @@ private void jButtonAddActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FI
+            if (this.tryParsing(input)) {
+                structure.add(Integer.parseInt(input));
+            } else {
+                return;
```
Instead of returning, how about we give a message to the user using JOptionPane giving feedback on sending in an integer only.
I dont think return is necessary here, is it?

alanefl replied:
The return is not necessary unless calling display() after a parsing failure is a problem, but I don't think it is.  Will replace with the JOptionPane notification that was in the "tryParsing" method before.

FOSSLGYS reviewed (commented):
Please update or share feedback on the raised points.

alanefl committed:
Addressing review comments.

Renamed the "tryParsing" method to "isInteger", pulled JOptionPane
notification outside the "tryParsing" method, and added new constant for
the non-integer input notification text.

alanefl committed:
Merge remote-tracking branch 'refs/remotes/FOSSLGYS/master'
